components/count-badge/count-badge.js

```diff
@@ -51,6 +51,33 @@ class CountBadge extends RtlMixin(LitElement) {
 				reflect: true,
 				attribute: 'hide-zero'
 			},
+			/**
+			 * Optionally add a description that will show as an aria-label on the badge.
+			 * @type {string}
+			 */
+			description: {
+				type: String,
+				reflect: true,
+				attribute: 'description'
+			},
+			/**
+			 * Optionally choose to add a tab stop to the badge. Defaults to false.
+			 * @type {boolean}
+			 */
+			tabStop: {
+				type: Boolean,
+				reflect: true,
+				attribute: 'tab-stop'
+			},
+			/**
+			 * Optionally choose to announce changes to the badge with an aria-live region. If the number property is changed, the description will be read by screenreaders. Defaults to false.
+			 * @type {boolean}
+			 */
+			announceChanges: {
+				type: Boolean,
+				reflect: true,
```

Comment:
I've noticed recently a non-zero performance cost to reflecting properties to attributes. So likely if you don't need a CSS selector, we could leave reflection off.